tools/lkl/tests/disk.sh

```diff
@@ -9,6 +9,8 @@ function prepfs()
     set -e
 
     file=`mktemp`
```

Comment:
We could use something like file=`mktemp disk-XXXX` to create the image file in the current directory

Reply:
thanks, modified as you suggested.